### DISCUSSION TITLE
Where can I find API docs for Whisper?

### DISCUSSION THREAD

**Discussion by blackears** [openai/whisper#2050] [Q&A]
Is there any documentation for Whisper's API?  All I can find is the example code on the main page.

**Comment by phineas-pta** [comment]
u mean openai sppech to text api ?

**Comment by blackears** [comment]
Yes.  Something that lists all the Python object methods.

**Reply by phineas-pta** [reply]
that's the opposite of what i said

anyway see purfview comment

**Comment by Purfview** [comment]
Read here:

https://github.com/openai/whisper/blob/ba3f3cd54b0e5b8ce1ab3de13e32122d0d5f98ab/whisper/transcribe.py#L59-L114

And here:

https://github.com/openai/whisper/blob/ba3f3cd54b0e5b8ce1ab3de13e32122d0d5f98ab/whisper/transcribe.py#L513-L548
